### PR TITLE
fix(route): 修复中国传媒大学研究生招生网

### LINKED ISSUE #10766: 中国传媒大学研究生招生网抓取失败
### 路由地址

```routes
/cuc/yz
```

### 完整路由地址

```fullroutes
/cuc/yz
```

### 相关文档

https://docs.rsshub.app/university.html#zhong-guo-chuan-mei-da-xue

### 预期是什么？

可以正常抓取到内容

### 实际发生了什么？

Route requested: /cuc/yz
Error message: The "url" argument must be of type string. Received undefined
Helpful Information to provide when opening issue:
Path: /cuc/yz
Node version: v16.17.0
Git Hash: 7ef81ae

### 部署

RSSHub 演示 (https://rsshub.app)

### 部署相关信息

_No response_

### 额外信息

```shell
该路由源码中的提取url的方法抛出异常
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

### PR DESCRIPTION
**Pull request by niuyi1017**
## 该 PR 相关 Issue / Involved issue

Close #10766 

## 完整路由地址 / Example for the proposed route(s)

```routes
/cuc/yz
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [ x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/router.js [hunk 1]
```diff
@@ -597,7 +597,7 @@ router.get('/cas/iee/kydt', lazyloadRouteHandler('./routes/universities/cas/iee/
 router.get('/cas/cg/:caty?', lazyloadRouteHandler('./routes/universities/cas/cg/index'));
 
 // 中国传媒大学
-router.get('/cuc/yz', lazyloadRouteHandler('./routes/universities/cuc/yz'));
+router.get('/cuc/yz', lazyloadRouteHandler('./v2/cuc/yz'));
```
Comment (TonyRL) [inline lib/router.js:600]: Please comment this.

File: lib/routes/universities/cuc/yz.js [hunk 2]
```diff
@@ -18,8 +18,7 @@ module.exports = async (ctx) => {
     const $ = cheerio.load(responseHtml);
 
     const notice = $('#notice_area').children('.notice_block_top');
-    const content = notice.children('.notice_content1').children();
-
```
Comment (TonyRL) [inline lib/routes/universities/cuc/yz.js:22]: Please remove those file.

File: lib/v2/cuc/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'yz.cuc.edu.cn': {
```
Comment (TonyRL) [inline lib/v2/cuc/radar.js:2]:
```suggestion
    'cuc.edu.cn': {
```

File: lib/v2/cuc/yz.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    const responseHtml = iconv.decode(response.data, 'gbk');
+    const $ = cheerio.load(responseHtml);
+
+    const notice = $('#notice_area').children('.notice_block_top');
+    const content = notice.children('.notice_content1').children('p');
+    const items = content
+        .map((_, elem) => {
+            const a = $('a', elem);
+            return {
+                link: url.resolve(host, a.attr('href')),
```
Comment (TonyRL) [inline lib/v2/cuc/yz.js:26]: URL.resolve is deprecated. please use wga

Comment (niuyi1017) [reply lib/v2/cuc/yz.js:26]: I haven`t use wga,could you please to show me the doc`s url?

File: lib/v2/cuc/radar.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+        _name: '中国传媒大学',
+        www: [
```
Comment (TonyRL) [inline lib/v2/cuc/radar.js:4]:
```suggestion
        yz: [
```

File: lib/v2/cuc/yz.js (hunk 4 above)
Comment (TonyRL) [reply lib/v2/cuc/yz.js:26]: "URL.resolve is deprecated. please use whatwg API instead"
https://nodejs.org/api/url.html

File: lib/v2/cuc/yz.js [hunk 6]
```diff
@@ -18,13 +26,12 @@ module.exports = async (ctx) => {
...
     const notice = $('#notice_area').children('.notice_block_top');
-    const content = notice.children('.notice_content1').children();
-
+    const content = notice.children('.notice_content1').children('p');
     const items = content
         .map((_, elem) => {
             const a = $('a', elem);
             return {
-                link: url.resolve(host, a.attr('href')),
+                link: resolve(host, a.attr('href')),
```
Comment (TonyRL) [inline lib/v2/cuc/yz.js:34]: simply
```suggestion
                link: new URL(a.attr('href'), host).href,
```
is enough

Comment (niuyi1017) [reply lib/v2/cuc/yz.js]: Thank you for your patience

**Pull request merged**